aspnetcore/signalr/version-differences.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Differences between SignalR and SignalR Core
+author: rachelappel
+description: Differences between SignalR and SignalR Core
+manager: wpickett
+monikerRange: '>= aspnetcore-2.1'
+ms.author: rachelap
+ms.date: 06/30/2018
+ms.prod: asp.net-core
+ms.technology: aspnet
+ms.topic: article
+uid: signalr/version-differences
+---
+
+# Differences between SignalR and ASP.NET Core SignalR
+
+ASP.NET Core SignalR is not compatible with clients or servers for ASP.NET SignalR. This article details the features which have been removed or changed in the ASP.NET Core SignalR.
+
+## Feature differences
+
+### Automatic reconnects
+
+Automatic reconnects are no longer supported. Previously, SignalR tried to reconnect to the server if the connection was dropped. Now, if the client is disconnected the user must explicitly start a new connection if they want to reconnect.
+
+### Protocol support
+
+ASP.NET Core SignalR supports JSON, as well as a new binary protocol based on MessagePack [MessagePack](xref:signalr/messagepackhubprotocol). Additionally, custom protocols can be created.
+
+## Differences on the server
+
+The SignalR server-side libraries are included in the `Microsoft.AspNetCore.App` package that is part of the **ASP.NET Core Web Application** template for both Razor and MVC projects.
+
+SignalR is an ASP.NET Core middleware, so it must be configured by calling `AddSignalR` in `Startup.ConfigureServices`.
+
+```csharp
+services.AddSignalR();
+```
+
+To configure routing, map routes to hubs inside the `UseSignalR` method call in the `Startup.Configure` method.
+
+```csharp
+app.UseSignalR(routes =>
+{
+    routes.MapHub<ChatHub>("/hub");
+});
+```
+
+### Sticky sessions now required
+
+Because of how scale-out worked in the previous versions of SignalR, clients could reconnect and send messages to any server in the farm. Due to changes to the scale-out model, as well as not supporting reconnects, this is no longer supported. Now, once the client connects to the server it needs to interact with this server for the duration of the connection.
```

Comment:
it needs to interact with ~~this~~ **the same** server for the duration of the connection.